.github/CODEOWNERS

```diff
@@ -144,6 +144,7 @@ packages/react-spinner/ @microsoft/cxe-red @tomi-msft
 packages/react-switch/ @microsoft/cxe-red @behowell @khmakoto @ecraig12345 @czearing
 packages/react-tabs/ @microsoft/cxe-coastal @geoffcoxmsft
 packages/react-text/ @microsoft/cxe-prg
+packages/react-toolbar/ @microsoft/cxe-prg @chpalac @ling1726
```

Comment:
Should this be owned by @microsoft/teams-prg? We had toolbar as part of the Menu group of components owned by teams-prg. We don't have to block on figuring this out though, we can always update it later